editor/plugins/script_editor_plugin.cpp

```diff
@@ -261,37 +261,6 @@ class EditorScriptCodeCompletionCache : public ScriptCodeCompletionCache {
 		}
 	}
 
-	virtual RES get_cached_resource(const String &p_path) {
```

Comment:
For the reference, this is used in `3.2` by GDScript:
```
modules/gdscript/gdscript_parser.cpp
508:                                            res = ScriptCodeCompletionCache::get_singleton()->get_cached_resource(path);
518:                                            res = ScriptCodeCompletionCache::get_singleton()->get_cached_resource(path);

modules/gdscript/gdscript_editor.cpp
884:                                                                                                                                    scr = ScriptCodeCompletionCache::get_singleton()->get_cached_resource(script);
```

But indeed GDScript 2.0 does not seem to rely on it anymore. CC @vnen just in case, but should be safe to remove.

Reply:
It's an oversight in my part, this should still be used: https://github.com/godotengine/godot/blob/edb3686ee2379ad28d1273797db8bd9ded38ff1e/modules/gdscript/gdscript_analyzer.cpp#L2429

(I guess I need to go over all the TODOs later)

Reply:
Ok, re-added.